astronomy-pipeline/Dockerfile

```diff
@@ -0,0 +1,19 @@
+
+FROM public.ecr.aws/lambda/python:latest
+
+WORKDIR ${LAMBDA_TASK_ROOT}
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+
+COPY astronomy_load.py .
+COPY astronomy_extract.py .
+COPY astronomy_extract_functions.py .
+COPY astronomy_transform.py .
+COPY astronomy_transform_functions.py .
+COPY astronomy_pipeline.py .
```

Comment:
This should be changed in a future pull request

Reply:
New change has been discussed and should be solved by submitted request.